src/Root.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {
+  Navigate,
+  Route,
+  HashRouter as Router,
+  Routes,
+} from 'react-router-dom';
+
+import { App } from './App';
+import { HomePage } from './pages/HomePage';
+import { PeoplePage } from './pages/PeoplePage/PeoplePage';
+import { NotFoundPage } from './pages/NotFoundPage';
+
+export const Root: React.FC = () => (
+  <Router>
+    <Routes>
+      <Route path="/" element={<App />}>
```

Comment:
You can add all paths in the enum and use them in all places in your application.